### PR TITLE
Add Condition widget test

### PR DESCRIPTION
**Pull request by scottbell**
### All Submissions:

* [X] Have you followed the guidelines in our [Contributing document](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md)?
* [X] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/nasa/openmct/pulls) for the same update/change?
* [X] Is this change backwards compatible? Will users need to change how they are calling the API, or how they've extended core plugins such as Tables or Plots?

### Author Checklist

* [X] Changes address original issue?
* [X] Unit tests included and/or updated with changes?
* [X] Command line build passes?
* [X] Has this been smoke tested?
* [X] Testing instructions included in associated issue?

### CODE REVIEW

File: src/utils/testing.js [hunk 1]
```diff
@@ -283,6 +283,16 @@ function setMockObjects() {
                 composition: [],
                 location: "mine"
             },
+            conditionWidget: {
```
Comment (nikhilmandlik) [inline src/utils/testing.js:286]: wondering if this is correct place to add mock data (unless used by multiple plugins/components).
In this case I think mock data is specific to `conditionWidget`, I guess we can define it in conditionWidget spec file?

**Review by nikhilmandlik — CHANGES_REQUESTED** [review]
Looks good just one small change / suggestion.

**Comment by akhenry** [comment]
Something kinda wild going on with this PR - 88 files changed.Is this based on the 1.7.8 branch? Might need to be rebased over master.

**Comment by scottbell** [comment]
> Something kinda wild going on with this PR - 88 files changed.Is this based on the 1.7.8 branch? Might need to be rebased over master.

Yes, before changing the branch target, it had a lot fewer changed files!

**Pull request closed without merging**